snippets/getCookie.md

```diff
@@ -0,0 +1,21 @@
+### getCookie
+
+Get a cookie by name in browser environment
+
+Use `document.cookie` to get all cookies.
+Use `String.prototype.split('; ')` to separate cookie name from cookie value.
+Use `if (parts.length === 2)` to check if exists any value.
```

Comment:
This could be added in the above line with a comma, without documenting the `if` part, it's very straightforward.